.github/workflows/test.yml

```diff
@@ -12,14 +12,17 @@ on:
       - 'releases/*'
 
 jobs:
-  # test local kind start up
-  test_local_kind:
+  # test local cluster start up
+  test_local_cluster:
     runs-on: ubuntu-latest
     strategy:
       matrix:
         # I suppose we'd better update version pre kepler release.
         # keep from current default version and latest version.
+        platform: [kind, microshift]
         kubectl_version: [v1.25.4, v1.27.2]
+        # So far we just have a single version for microshift
+        # in the future, we should considering have a local test for each cluster providers
         kind_version: [0.17.0, 0.19.0]
```

Comment:
I still think we don't need to test different version that does not have major changes...

Reply:
I suppose the depends on cluster provider.
for example micro shift don't have any other label?https://quay.io/repository/microshift/microshift-aio?tab=tags&tag=latest
or say the label logic looks wired.

Reply:
We could just have:
```
 kubectl_version: [v1.27.2]
        # So far we just have a single version for microshift
        # in the future, we should considering have a local test for each cluster providers
        kind_version: [0.19.0]
```